### DISCUSSION TITLE
Installation in Arduino IDE

### DISCUSSION THREAD

**Discussion by flpxra** [SpenceKonde/DxCore#327] [Q&A] [answered]
Is there a URL link with a JSON file to install DxCore via Boards Manager in the Arduino IDE?
When I install it manually, it always gives me an error compiling the code: panic: no major version found

**Comment by grandaspanna** [comment, marked as answer]
The install instructions here should help: https://github.com/SpenceKonde/DxCore/blob/master/Installation.md

**Reply by ds5ulo** [reply]
Index file C:\Users\gram\AppData\Local\Arduino15\package_drazzy.com_index.json Skipping, parsing error occurred:
com.fasterxml.jackson.databind.JsonMappingException: Unexpected character (']' (code 93)): expected a value
 at [Source: (FileInputStream); line: 3242, column: 12]
 at [Source: (FileInputStream); line: 3242, column: 11] (through reference chain: cc.arduino.contributions.packages.ContributionsIndex["packages"]->java.util.ArrayList[1]->cc.arduino.contributions.packages.Contrib

**Comment by SpenceKonde** [comment]
Use version 1.8.13 of the ide, that is the only working version. The bug is
caused by a regression in 1.8.14 which they show little inclination to fix.

____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore: Arduino support for almost every ATTiny microcontroller
Contact: ***@***.***

**Comment by flpxra** [comment]
Oh, I didn't notice that manual.
I used the installation by Boards Manager and everything works in Arduino IDE 1.19, the code compiles (I did not check the upload).
Additionally, I was able to install DxCore in Sloeber IDE 4.4.
@SpenceKonde you are doing a great job!

**Comment by SpenceKonde** [comment]
Wish I felt that way.

1.8.13 is the only version of Arduino IDE that I consider working
currently,.

Andwhat the fuck is slober IDE? There's PIO, there's that unspellable
thing, there's 50 million others, and everyone expects my cores to work on
them and asks questions and how the fuck shoudl I know about something an
ide I never heard before.

If I can figure out a way to do it, it I will make this core ONLY work on
IDEs for while a knowledgable user has written a basic guide. Right now
>50% of our users are on other IDES tjat I knnow nothing about. And I my
group of real life friends just shattererd.

On Wed, Aug 17, 2022 at 3:08 AM flexphperia ***@***.***>
wrote:

> Oh, I didn't notice that manual.
> I used the installation by Boards Manager and everything works in Arduino
> IDE 1.19, the code compiles (I did not check the upload).
> Additionally, I was able to install DxCore in Sloeber IDE 4.4.
> @SpenceKonde <https://github.com/SpenceKonde> you are doing a great job!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SpenceKonde/DxCore/discussions/327#discussioncomment-3412098>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABTXEW3UWYSYUCRYUURXJBLVZSFWTANCNFSM56YGGFPQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore <https://github.com/SpenceKonde/ATTinyCore>: Arduino support for
all pre-2016 tinyAVR with >2k flash!
megaTinyCore <https://github.com/SpenceKonde/megaTinyCore>: Arduino support
for all post-2016 tinyAVR parts!
DxCore <https://github.com/SpenceKonde/DxCore>: Arduino support for the AVR
Dx-series parts, the latest and greatest from Microchip!
Contact: ***@***.***

**Comment by ObviousInRetrospect** [comment]
why are you wanting to install manually?  AFAIK the head of tree is not something you likely want to use right now.
